### PR TITLE
All Content, Sections and Section page

### PR DESCRIPTION
**Pull request by YoHype**
### Types of changes

- [ ] Fix (non-breaking change that fixes existing issue)
- [ ] Feature (non-breaking change that adds a feature)
- [ ] Breaking change (fix or feature that breaks or removes other functionality)

### Pre-flight checklist

- [ ] PR changes requires documentation change
  - [ ] I have updated the documentation accordingly
- [ ] I have tested my change, and run all unit tests 
- [ ] I ran ``dotnet format`` and committed+pushed the result

### Fixes issues:

- [x] #[some issue]

## Screenshots of the solution (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ export connectionString=$(echo "server=localhost;database=${database};user id=sa
 #### Enable User Secrets
 ```powershell
 dotnet user-secrets init --project SELearning.API
-dotnet user-secrets set "ConnectionStrings:SELearning" "$connectionString" --project SELearning.API
+dotnet user-secrets set "ConnectionStrings:SELearning" "$connectionstring" --project SELearning.API
```
Comment (avborup) [inline README.md:31]: This change should be removed.

File: SELearning/Shared/Section/Sections.razor [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    protected async override void OnInitialized()
+    {
+        await RetrieveSections();
+    }
+
+    private async void OnSectionDeleted(Section sec)
+    {
+        System.Console.WriteLine($"Comment from {sec.Id} is deleted!");
+        @* _sections.Remove(sec); *@
+    }
```
Comment (avborup) [inline SELearning/Shared/Section/Sections.razor:31]: There are warnings here that should be fixed

**Review by avborup — CHANGES_REQUESTED** [review]

**Comment by avborup** [comment]
I also get a 404 when attempting to delete content:

image

**Comment by avborup** [comment]
The sections list shows the ID instead of the name:

image

**Comment by avborup** [comment]
Nothing happens when pressing delete section

**Comment by avborup** [comment]
Content list does not update after delete

**Comment by avborup** [comment]
Getting content in section shows nothing

**Review by Arneproductions — APPROVED** [review]

**Review by JoachimBorup — APPROVED** [review]

**Review by avborup — APPROVED** [review]

**Pull request merged**